cdap-ui/app/directives/log-viewer-preview/log-viewer-preview.html

```diff
@@ -0,0 +1,215 @@
+          <div class="status-circle" ng-class="{'program-primary-circle' : LogViewerPreview.statusType === 0, 'program-secondary-circle' : LogViewerPreview.statusType === 1, 'program-completed-circle' : LogViewerPreview.statusType === 2, 'program-default-circle' : LogViewerPreview.statusType === 3}">
+          </div>
+          {{LogViewerPreview.programStatus}}
+        </span>
+      </div>
+      <div class="log-stats-total log-stats text-center">
+          <span class="stat-name">Total Messages</span>
+          <div class="stat-count total-num">
+            <span class="stat-count-number">{{LogViewerPreview.totalCount}}</span></div>
+      </div>
+      <div class="log-stats-errors log-stats text-center">
+        <span class="stat-name log-stats-errors-text">Error</span>
+          <div class="stat-count errors-num"><span class="stat-count-number">{{LogViewerPreview.errorCount}}</span></div>
+      </div>
+      <div class="log-stats-warnings log-stats text-center">
+        <span class="stat-name log-stats-warnings-text">Warning</span>
+        <div class="stat-count warnings-num">
+          <span class="stat-count-number">{{LogViewerPreview.warningCount}}</span>
+        </div>
+      </div>
+    </div>
+    <div class="log-actions">
+      <div class="log-action-container">
+        <a href="{{LogViewerPreview.getDownloadUrl('raw')}}"
+          target="_blank"
+          class="log-action-btn view-raw btn btn-default"
+          ng-disabled="!LogViewerPreview.displayData.length || LogViewerPreview.loading"
+          ng-click="LogViewerPreview.preventClick($event)">
+          <span>View Raw Logs</span>
+        </a>
+
+        <a href="{{LogViewerPreview.getDownloadUrl('download')}}"
+          target="_blank"
+          class="log-action-btn download-all btn btn-default"
+          ng-disabled="!LogViewerPreview.displayData.length || LogViewerPreview.loading"
+          ng-click="LogViewerPreview.preventClick($event)">
+          <span>Download All</span>
+        </a>
+        <div class="search">
+          <input type="text" class="log-searchbar" placeholder="Search" ng-model="LogViewerPreview.searchText" ng-model-options="{ debounce: 500 }" ng-change="LogViewerPreview.filterSearch()">
+          <span class="fa fa-search"></span>
+        </div>
+        <i class="fa fa-expand" aria-hidden="true" ng-click="LogViewerPreview.updateScreenChangeInStore(!LogViewerPreview.fullScreen)"></i>
+      </div>
+    </div>
+  </div>
+
+  <a class="sr-only" id="logs-export-link" href="{{LogViewerPreview.url}}" download="{{LogViewerPreview.exportFileName}}.log">Export</a>
+
+  <!-- LOGS TIMELINE
+  <my-timeline
+    namespace-id="{{LogViewerPreview.namespaceId}}"
+    app-id="{{LogViewerPreview.appId}}"
+    program-type="{{LogViewerPreview.programType}}"
+    program-id="{{LogViewerPreview.programId}}"
+    run-id="{{LogViewerPreview.runId}}">
+  </my-timeline>
+  -->
```

Comment:
just remove this. We can just copy paste from the other log viewer when we have support from backend